application-development/helloworld-springboot-auth/src/main/resources/application.yml

```diff
@@ -0,0 +1,16 @@
+security:
+  oauth2:
+    client:
+      clientId: 667713450471-6u065o1ph801fr80kqa0popiudsvathb.apps.googleusercontent.com
+      clientSecret: cqR_pUlx1BjEBumoE0o69WkF
```

Comment:
Should we remove this and just say <INSERT_YOUR_CLIENT_ID HERE>? or something? 

I assume that including this lets them run locally without creating a client id, but they will be unable to deploy and get the "invalid redirect url", right? 
I don't have a strong opinion.

Reply:
Thanks for catching this... I thought I had cleaned this up